editor/block-switcher/index.js

```diff
@@ -113,14 +118,20 @@ function BlockSwitcher( { block, onTransform } ) {
 }
 
 export default connect(
-	( state, ownProps ) => ( {
-		block: getBlock( state, ownProps.uid ),
-	} ),
-	( dispatch, ownProps ) => ( {
-		onTransform( block, name ) {
+	( state, ownProps ) => {
+		const isMultiBlock = ! ownProps.uid && getSelectedBlockCount( state ) > 1;
+		const blocks = isMultiBlock ? getMultiSelectedBlocks( state ) :
```

Comment:
It feels a bit weird to retrive the blocks from the state when there's a multiselection and retrieve them from the `uid` when passed. Can we update the component to always use a `uids` prop To make this consistent?